### ISSUE TITLE
Context switch is not working - possible clash with same context name value

### ISSUE THREAD

**Issue by vinibali** [danielfoehrKn/kubeswitch#137]
Hi!
I have some kubeconfigs which has the same context name inside. Even kubectl is not able identify both if these are listed in the KUBECONFIG env variable, like
```
KUBECONFIG=/home/user/.kube/cluster1.conf:/home/user/.kube/cluster2.conf
```
Is there some other way to identify the kubeconfigs with the context-cluster key?
Unfortunately I couldn't rename the context, because that breaks the functionality of the config.
I experience pretty much the same with kubectx:
https://github.com/ahmetb/kubectx/issues/428

**Comment by danielfoehrKn** [comment]
Try to put the kubeconfigs under different parent filesystem paths (directory).

```
KUBECONFIG=/home/user/.kube/parent1/cluster1.conf:/home/user/.kube/parent2/cluster2.conf
```

**Comment by vinibali** [comment]
that's not working unfortunately, that's still one context.

**Comment by danielfoehrKn** [comment]
Do you have https://github.com/danielfoehrKn/kubeswitch/blob/master/types/config.go#L116 set to `true`? Then for both kubeconfigs, it should show the distinct parent directory followed by the same context name.

**Comment by vinibali** [comment]
Yes, that's true by default. As it turned out this is not working, I even tried to set the default setting here:
```
$ cat switch-config.yaml
kind: SwitchConfig
version: v1alpha1
showPrefix: true
$ echo $KUBECONFIG
KUBECONFIG=/home/user/.kube/cluster1.conf:/home/user/.kube/cluster2.conf
```
The only way, where the different config directories was shown, when I utilized the filesystem kind:
```
$ cat switch-config.yaml
kind: SwitchConfig
version: v1alpha1
kubeconfigStores:
- kind: filesystem
  kubeconfigName: "cluster*.conf"
  showPrefix: true
  showPreview: false
  paths:
    - ~/.kube/parent1
    - ~/.kube/parent2
```
In this case I could see 4 contexts, but 3 had the same preview and 1 was the different one. What I can see now it's not working if you are using the KUBECONFIG with the slimmed down switch-config.
The preview supposed to work if the KUBECONFIG environment variable is unset and the filesystem kind identifies the configs in their own directory. Unfortunately in this case I don't know how how to target configs/contexts as kubectl is looking for the K8s cluster on the local machine.

Sorry for the late reply
